packages/gatsby-remark-code-repls/README.md

```diff
@@ -42,30 +42,59 @@ to HTML links that open the embedded code examples in a REPL. For example:
 
 ```html
 <!-- before -->
-[See it in Babel](babel://hello-world)
+[See it in Babel](babel://hello-world.js)
 
 <!-- after -->
 <a href="https://babeljs.io/repl/#?presets=react&code_lz=...">
   See it in Babel
 </a>
 
 <!-- before -->
-[Try it on CodePen](codepen://components-and-props/rendering-a-component)
+[Try it on CodePen](codepen://components-and-props/rendering-a-component.js)
 
 <!-- after -->
 <a href="/redirect-to-codepen/components-and-props/rendering-a-component">
   Try it on CodePen
 </a>
 
 <!-- before -->
-[Try it on CodeSandbox](codesandbox://components-and-props/rendering-a-component)
+[Try it on CodeSandbox](codesandbox://components-and-props/rendering-a-component.js)
```

Comment:
Let's tell people to add explicit file extensions now that we support multiple types of files, even if we have a fallback mechanism for JavaScript files.

Reply:
I think this is a good suggestion.